app/Support/MaintenanceService.php

```
@@ -306,6 +318,165 @@ public function runNotifications(): array
+                ");
+                $repair->bind_param('i', $loanId);
+                $repair->execute();
+                $changed = $repair->affected_rows;
+                $repair->close();
+                if ($changed !== 1) {
+                    $this->db->rollback();
+                    continue;
+                }
+
+                if (!$integrity->recalculateBookAvailability($bookId, insideTransaction: true)) {
+                    throw new \RuntimeException('Failed to recalculate availability while repairing a ready pickup.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '458', column '31'). (undefined)

(MissingImport)